GameData/RealismOverhaul/Engine_Configs/Raptor.cfg

```diff
@@ -7,40 +7,42 @@
 //	Raptor
 //	Superheavy
 //
-//	Dry Mass: ??? Kg
-//	Thrust (SL): ??? kN
-//	Thrust (Vac): 1779.29 kN
-//	ISP: 332 SL / 356 Vac
+//	Dry Mass: 1500 Kg ~ Goal
+//	Thrust (SL): 2241.90 kN
+//	Thrust (Vac): ???
+//	ISP: 330 SL / 355 Vac
 //	Burn Time: ???
-//	Chamber Pressure: 27 MPa
+//	Chamber Pressure: 33 MPa
 //	Propellant: LOX / CH4
-//	Prop Ratio: 3.8
-//	Throttle: 20% - 100%
-//	Nozzle Ratio: 35
-//	Ignitions: ???
+//	Prop Ratio: 3.55
+//	Throttle: 40% - 100%
+//	Nozzle Ratio: 40
+//	Ignitions: Almost Unlimited due to Torch-lit System
 //	=================================================================================
 //	Raptor Vac
 //	Starship
 //
-//	Dry Mass: ??? Kg
-//	Thrust (SL): ??? kN
-//	Thrust (Vac): 1957.22 kN
-//	ISP: ??? SL / 379 Vac
+//	Dry Mass: 200 TWR Ratio ~ Goal
+//	Thrust (SL): ???
+//	Thrust (Vac): 2241.90 kN ~ Guess
+//	ISP: 350 SL / 380 Vac
 //	Burn Time: ???
-//	Chamber Pressure: 27 MPa
+//	Chamber Pressure: 33 MPa
 //	Propellant: LOX / CH4
-//	Prop Ratio: 3.8
-//	Throttle: 20% - 100%
-//	Nozzle Ratio: 161
-//	Ignitions: ???
+//	Prop Ratio: 3.55
+//	Throttle: 40% - 100%
+//	Nozzle Ratio: 107 ~ Not Sure if its the same as expansion ratio?
```

Comment:
Should be the same thing, yeah. It's the ratio between the nozzle's exit area and its throat section.